README.md

```diff
@@ -47,26 +47,19 @@ npm start
 
 #### Trading
 
-* `CONNECTOR_LEDGERS` (default: `[]`) Ledgers where this connector has accounts. Used to auto-generate `CONNECTOR_PAIRS`.
-```js
-[
-  "USD@example.usd-ledger.",
-  "EUR@example.eur-ledger."
-]
-```
-* `CONNECTOR_CREDENTIALS` (default: `{}`) Connector's login credentials for various ledgers, ex.
+* `CONNECTOR_LEDGERS` (default: `{}`) Connector's login credentials for ledgers where it has accounts. Used to auto-generate `CONNECTOR_PAIRS`.
 ```js
 {
   // Using Basic Auth
-  "<ledger_address>": {
+  "USD@example.usd-ledger.": {
```

Comment:
Oh, it should probably be made clearer that those maps are going to be passed directly to the plugins. Another way to handle this would be:

```js
"ledger.address.": {
  "currency": "USD",
  "plugin": "ilp-plugin-bells",
  "options": {
    // options passed directly to the plugin
  }
}
```
An advantage of this would be it would be easier to add ledger-specific configuration that is used by the connector but not the plugin.